### PR TITLE
Add documents on "HSF Affiliated Projects and Software" and related Guidelines

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Document as finalised by Pere and I with community input, and agreed within the SG.

I started a new folder for this, which should become the one for the top-level menu "Projects & Support" on the homepage.

### CODE REVIEW

**Comment by eduardo-rodrigues** [comment]
This is now ready. The only things missing are the links for the AG (to be added once available) and the documents on the guidelines, which will need to be finalised soon and put online as well.

I'm not suggesting that anyone reviews at this time on a Friday. Let's finalise next week.

**Comment by eduardo-rodrigues** [comment]
Kind ping for 1-2 of you to review this MR :-).

File: projects/affiliated.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+layout: plain
+---
+
+HSF Affiliated Projects and Software
+
+## Definitions
+
+*HSF Affiliated Projects and Software* are community-driven and community-oriented "endeavours" or "products" of wide and recognised interest and applicability beyond a single collaboration or experiment.
+
+*HSF Affiliated Projects and Software* are community projects or software packages with person-power, and possibly dedicated funding, which connect strongly to the HSF and align with [its remit]({{ site.baseurl }}/organization/goals.html). They benefit from their inclusion in, or association with, the HSF through access to the community for wide visibility and easier promotion through the network. Affiliation makes it clear that the HSF has no control over the endeavours/products, hence no responsibility for evolution or maintenance.
```
Comment (hegner) [inline projects/affiliated.md:12]: wouldn't use the word remit and just talk about goals.

File: projects/affiliated.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+In broad terms, the attribution of the endorsement level is based on the following guidelines, which are detailed in the dedicated document on Guidelines for HSF Affiliated Projects and Software, and Endorsement Badge Levels.
+* Bronze: new or young endeavour, likely evolving from and within a collaboration or experiment, but with the potential for other communities to use. It should be committed to meeting best practices in software engineering, e.g., documentation and a test suite.
+* Silver: aiming for gold but in an earlier phase towards strong community support (e.g., adoption is still relatively shy, maintenance is not secured at least in the medium term by more than a single person). High standards of software engineering should be met.
+* Gold: endeavour adopted by several collaborations and/or experiments with a strong and long-term community support model.
+
+The HSF envisages a scheme for Annual Project Awards. A concrete proposal will be presented to the HSF Advisory Group once the latter is put in place.
+
+## Best-practice guidelines
+
+They are detailed in the document Guidelines for HSF (Affiliated) Projects and Software.
```
Comment (hegner) [inline projects/affiliated.md:37]: why "affiliated" in brackets?

File: projects/affiliated.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+## HSF "Reviews"
```
Comment (hegner) [inline projects/affiliated.md:39]: No quotation marks - that are genuine reviews, no?

**Review by hegner — CHANGES_REQUESTED** [review]
Just minor points. Thank you!

File: projects/affiliated.md (hunk 2 above)
Comment (eduardo-rodrigues) [reply projects/affiliated.md:37]: Oh, that's a left-over from the times we made a distinction. Thanks for spotting it. Removing the parentheses.

File: projects/affiliated.md (hunk 1 above)
Comment (eduardo-rodrigues) [reply projects/affiliated.md:12]: OK.

Comment (eduardo-rodrigues) [inline projects/affiliated.md:12]:
```suggestion
*HSF Affiliated Projects and Software* are community projects or software packages with person-power, and possibly dedicated funding, which connect strongly to the HSF and align with [its goals]({{ site.baseurl }}/organization/goals.html). They benefit from their inclusion in, or association with, the HSF through access to the community for wide visibility and easier promotion through the network. Affiliation makes it clear that the HSF has no control over the endeavours/products, hence no responsibility for evolution or maintenance.
```

File: projects/affiliated.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+* Silver: aiming for gold but in an earlier phase towards strong community support (e.g., adoption is still relatively shy, maintenance is not secured at least in the medium term by more than a single person). High standards of software engineering should be met.
+* Gold: endeavour adopted by several collaborations and/or experiments with a strong and long-term community support model.
+
+The HSF envisages a scheme for Annual Project Awards. A concrete proposal will be presented to the HSF Advisory Group once the latter is put in place.
+
+## Best-practice guidelines
+
+They are detailed in the document Guidelines for HSF Affiliated Projects and Software.
+
+## HSF "Reviews"
```
Comment (eduardo-rodrigues) [inline projects/affiliated.md:39]:
```suggestion
## HSF Reviews
```

**Comment by eduardo-rodrigues** [comment]
Thanks. Approve when happy. I will wait for a second approval given the importance of the document.

**Comment by hegner** [comment]
@eduardo-rodrigues - would have suggested the same - another pair of eyes would be good. I am fine with the current state

**Review by hegner — APPROVED** [review]

**Review by graeme-a-stewart — CHANGES_REQUESTED** [review]
Thank you @eduardo-rodrigues - I made some suggestions here.

Before we accept it seems we need:

- The guidelines document
- Updated `navbar.ext` that links to these documents

**Comment by graeme-a-stewart** [comment]
> This is now ready. The only things missing are the links for the AG (to be added once available) 

It will be `/organization/advisory-group.html` - see #1597.

**Comment by graeme-a-stewart** [comment]
Sorry it took a while to review @eduardo-rodrigues - JuliaHEP was really full-on last week!

File: projects/affiliated.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+*HSF Affiliated Projects and Software* are community-driven and community-oriented "endeavours" or "products" of wide and recognised interest and applicability beyond a single collaboration or experiment.
+
+*HSF Affiliated Projects and Software* are community projects or software packages with person-power, and possibly dedicated funding, which connect strongly to the HSF and align with [its goals]({{ site.baseurl }}/organization/goals.html). They benefit from their inclusion in, or association with, the HSF through access to the community for wide visibility and easier promotion through the network. Affiliation makes it clear that the HSF has no control over the endeavours/products, hence no responsibility for evolution or maintenance.
+
+Projects are in general not hosted on the HSF GitHub repository since they are externally managed and maintained, though some are on the [HSF GitHub repository](https://github.com/HSF).
+
+## Engagement and endorsement
+
+Any member of the community is welcome to engage with the HSF in general and the
```
Comment (eduardo-rodrigues) [inline projects/affiliated.md:18]:
```suggestion
Anyone is welcome to engage with the HSF in general and the
```

File: projects/affiliated.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
+---
+title: "HSF Affiliated Projects and Software"
+layout: plain
+---
+
+HSF Affiliated Projects and Software
```
Comment (eduardo-rodrigues) [inline projects/affiliated.md:6]:
```suggestion
```

[the remark this answers is not in the document]
Comment (eduardo-rodrigues) [reply projects/affiliated.md]: OK, I did not realise how the templates work.

File: projects/affiliated.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (eduardo-rodrigues) [reply projects/affiliated.md]: OK

File: projects/affiliated.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+
+The list of HSF Affiliated Projects and Software is hosted in a dedicated area on the HSF website, curated and maintained by the HSF SG.
+
+The recognition of bijective engagement with the HSF is displayed via *GitHub Badges*.
+Three levels distinguish mainly the level of maturity, community support and  engagement: Bronze, Silver and Gold.
+In broad terms, the attribution of the endorsement level is based on the following guidelines, which are detailed in the dedicated document on Guidelines for HSF Affiliated Projects and Software, and Endorsement Badge Levels.
+* Bronze: new or young endeavour, likely evolving from and within a collaboration or experiment, but with the potential for other communities to use. It should be committed to meeting best practices in software engineering, e.g., documentation and a test suite.
+* Silver: aiming for gold but in an earlier phase towards strong community support (e.g., adoption is still relatively shy, maintenance is not secured at least in the medium term by more than a single person). High standards of software engineering should be met.
```
[the remark this answers is not in the document]
Comment (eduardo-rodrigues) [reply projects/affiliated.md]: OK

File: projects/affiliated.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+layout: plain
+---
+
+HSF Affiliated Projects and Software
+
+## Definitions
+
+*HSF Affiliated Projects and Software* are community-driven and community-oriented "endeavours" or "products" of wide and recognised interest and applicability beyond a single collaboration or experiment.
+
+*HSF Affiliated Projects and Software* are community projects or software packages with person-power, and possibly dedicated funding, which connect strongly to the HSF and align with [its goals]({{ site.baseurl }}/organization/goals.html). They benefit from their inclusion in, or association with, the HSF through access to the community for wide visibility and easier promotion through the network. Affiliation makes it clear that the HSF has no control over the endeavours/products, hence no responsibility for evolution or maintenance.
```
[the remark this answers is not in the document]
Comment (eduardo-rodrigues) [reply projects/affiliated.md:11]: You mean "HSF has no control" -> "HSF has no direct control"? Sure.

File: projects/affiliated.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+[HSF Steering Group (SG)]({{ site.baseurl }}/organization/team.html),
+in particular to discuss the route towards making a software project or package an HSF Affiliated Project or Software. This can be an informal discussion initially.
+
+Formal endorsement is provided by the HSF SG following a short evaluation period in consultation with the relevant HSF Working Group conveners and community experts. For transparency, these evaluations (start, scope and end) are advertised publicly via the HSF Forum, the HSF's main means of communication.
+
+The list of HSF Affiliated Projects and Software is hosted in a dedicated area on the HSF website, curated and maintained by the HSF SG.
+
+The recognition of bijective engagement with the HSF is displayed via *GitHub Badges*.
+Three levels distinguish mainly the level of maturity, community support and  engagement: Bronze, Silver and Gold.
+In broad terms, the attribution of the endorsement level is based on the following guidelines, which are detailed in the dedicated document on Guidelines for HSF Affiliated Projects and Software, and Endorsement Badge Levels.
```
[the remark this answers is not in the document]
Comment (graeme-a-stewart) [reply projects/affiliated.md:27]: Yeah, I agree about that. Let's try to get both done together.

File: projects/affiliated.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+title: "HSF Affiliated Projects and Software"
+layout: plain
+---
+
+
+## Definitions
+
+*HSF Affiliated Projects and Software* are community-driven and community-oriented "endeavours" or "products" of wide and recognised interest and applicability beyond a single collaboration or experiment.
+
+*HSF Affiliated Projects and Software* are community projects or software packages with person-power, and possibly dedicated funding, which connect strongly to the HSF and align with [its goals]({{ site.baseurl }}/organization/goals.html). They benefit from their inclusion in, or association with, the HSF through access to the community for wide visibility and easier promotion through the network. Affiliation makes it clear that the HSF has no control over the endeavours/products, hence no responsibility for evolution or maintenance.
```
Comment (eduardo-rodrigues) [inline projects/affiliated.md:11]:
```suggestion
*HSF Affiliated Projects and Software* are community projects or software packages with person-power, and possibly dedicated funding, which connect strongly to the HSF and align with [its goals]({{ site.baseurl }}/organization/goals.html). They benefit from their inclusion in, or association with, the HSF through access to the community for wide visibility and easier promotion through the network. Affiliation makes it clear that the HSF has no direct control over the endeavours/products, hence no responsibility for evolution or maintenance.
```

File: projects/affiliated.md [hunk 11]
```diff
@@ -0,0 +1,79 @@
+---
+title: "HSF Affiliated Projects and Software"
+layout: plain
+---
+
+HSF Affiliated Projects and Software
+
+## Definitions
```
[the remark this answers is not in the document]
Comment (eduardo-rodrigues) [reply projects/affiliated.md:7]: "Introduction" is just fine, yes :-).

File: projects/affiliated.md [hunk 12]
```diff
@@ -0,0 +1,78 @@
+---
+title: "HSF Affiliated Projects and Software"
+layout: plain
+---
+
+
+## Definitions
```
Comment (eduardo-rodrigues) [inline projects/affiliated.md:7]:
```suggestion
## Introduction
```

File: projects/affiliated.md [hunk 13]
```diff
@@ -0,0 +1,79 @@
...
+
+HSF Affiliated Projects and Software
+
+## Definitions
+
+*HSF Affiliated Projects and Software* are community-driven and community-oriented "endeavours" or "products" of wide and recognised interest and applicability beyond a single collaboration or experiment.
+
+*HSF Affiliated Projects and Software* are community projects or software packages with person-power, and possibly dedicated funding, which connect strongly to the HSF and align with [its goals]({{ site.baseurl }}/organization/goals.html). They benefit from their inclusion in, or association with, the HSF through access to the community for wide visibility and easier promotion through the network. Affiliation makes it clear that the HSF has no control over the endeavours/products, hence no responsibility for evolution or maintenance.
+
+Projects are in general not hosted on the HSF GitHub repository since they are externally managed and maintained, though some are on the [HSF GitHub repository](https://github.com/HSF).
```
[the remark this answers is not in the document]
Comment (eduardo-rodrigues) [reply projects/affiliated.md]: Good idea. Added.

**Comment by eduardo-rodrigues** [comment]
> > This is now ready. The only things missing are the links for the AG (to be added once available)
> 
> It will be `/organization/advisory-group.html` - see #1597.

Link added.

File: projects/affiliated.md [hunk 14]
```diff
@@ -0,0 +1,79 @@
+---
+title: "HSF Affiliated Projects and Software"
+layout: plain
+---
+
+HSF Affiliated Projects and Software
+
```
[the remark this answers is not in the document]
Comment (eduardo-rodrigues) [reply projects/affiliated.md:7]: This is a nice idea and I added your sentence verbatim.

File: projects/affiliated.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (eduardo-rodrigues) [reply projects/affiliated.md:32]: Added in the most recent commit.

File: projects/affiliated.md [hunk 15]
```diff
@@ -0,0 +1,79 @@
...
+In broad terms, the attribution of the endorsement level is based on the following guidelines, which are detailed in the dedicated document on Guidelines for HSF Affiliated Projects and Software, and Endorsement Badge Levels.
+* Bronze: new or young endeavour, likely evolving from and within a collaboration or experiment, but with the potential for other communities to use. It should be committed to meeting best practices in software engineering, e.g., documentation and a test suite.
+* Silver: aiming for gold but in an earlier phase towards strong community support (e.g., adoption is still relatively shy, maintenance is not secured at least in the medium term by more than a single person). High standards of software engineering should be met.
+* Gold: endeavour adopted by several collaborations and/or experiments with a strong and long-term community support model.
+
+The HSF envisages a scheme for Annual Project Awards. A concrete proposal will be presented to the HSF Advisory Group once the latter is put in place.
+
+## Best-practice guidelines
+
+They are detailed in the document Guidelines for HSF Affiliated Projects and Software.
```
[the remark this answers is not in the document]
Comment (eduardo-rodrigues) [reply projects/affiliated.md:42]: Added.

**Comment by eduardo-rodrigues** [comment]
Hi all. I believe I have now done everything expected :-). I hope I got all (cross-)links correct but good that a couple of us check ...

Related to this update, we need to discuss what exactly to do with the tab highlighted below:
image

For now I moved all 3 items to below the line, having the 2 new documents here above the line. This makes sense to me. This being said, the "Member projects" and "How to join as a project" need to be revamped or deleted. Shall we do that in a quick follow-up?

**Review by graeme-a-stewart — APPROVED** [review]

**Comment by graeme-a-stewart** [comment]
Thanks @eduardo-rodrigues - this is a lot of work!

I pushed a [commit](https://github.com/HSF/hsf.github.io/pull/1595/commits/c81fa5a7e2d22cf7c155e2a57560df1635c13373) that:

- Fixes navbar links
- Shortens the overlong titles
- Fixes a typo

If you are good with that change I think we can merge.

**Comment by eduardo-rodrigues** [comment]
Many thanks for the careful checks! I see 2 approvals, merging then :-).

**Pull request merged**